var/spack/repos/builtin/packages/py-mpi4py/package.py

```diff
@@ -15,10 +15,13 @@ class PyMpi4py(PythonPackage):
 
     pypi = "mpi4py/mpi4py-3.0.3.tar.gz"
     git = "https://github.com/mpi4py/mpi4py.git"
+    maintainers("dalcinl", "balay")
```

Comment:
@dalcinl Is it ok to add you to maintainers?

Reply:
No, I cannot take such responsibility, sorry. conda-forge maintenance already takes more bandwidth than the one I have available.

Reply:
ok reverted